benchmarks/filterapi.h

```diff
@@ -27,6 +27,11 @@
 #include "vqf_cpp.h"
 #include "simd-block.h"
 #endif
+#define __PF_AVX512__  (__AVX512CD__ & __AVX512DQ__ &__AVX512DQ__ &__AVX512DQ__)
+#ifdef __PF_AVX512__
+#include "prefix/min_pd256.hpp"
+#include "tcShortcut/TC-shortcut.hpp"
```

Comment:
Could we use lowercase file names? There are some file systems that are case insensitive, like https://discussions.apple.com/thread/251191099 - and I would like to avoid any troubles in this area.